style_editor.html

```diff
@@ -8841,6 +8849,7 @@
      if (bad_fps) bad_fps --;
   }
   if (bad_fps > 10 && use_graflex) {
+    FIND("error_message").innerHTML = "Struggling to render hilt model. Switching to simpler design.";
```

Comment:
This could be unhelpful since it might wipe out an existing error message that you actually wanted to see...

Reply:
ok. How about a popup?
![Screenshot 2023-05-28 at 12 20 05 AM](https://github.com/profezzorn/ProffieOS-StyleEditor/assets/53964195/fa0f1539-1ad7-4059-bc7d-b0a603076884)

Reply:
That's just annoying...
Maybe we can set the error message, but then set it back in a second if it's still the same?

Reply:
Or maybe if the popup had a "don't show this message again" checkbox?

Reply:
That's cool. However it would use the saveCookie function, which isn't coming until settings is checked in, so I'll popup for now.

Reply:
made into #97